### ISSUE TITLE
Too many Python scripts

### ISSUE THREAD

**Issue by pokemaniac05** [holzschu/a-shell#488]
I'm using an iPhone 8 running iOS 16.0.3 and latest a-shell version (build 270) from TestFlight.

If I try to run e.g. `pip install bpytop` or `pip install -r requirements.txt` (from a project with multiple dependencies), I get the following error in an alert window: `Too many Python scripts. Too many Python interpreters running at the same time. Try to close some of theme.` (I can't attach a screenshot because this iPhone isn't mine).

## How to reproduce this bug

Same as above or try to run the following commands:
`$ python`

`>>> import os`

`>>> os.system("python")`

Run again last two commands in the new interpreter and you'll get an error like this:
`Failed loading notAValidCommand from python3_
ios.framework/python3_ios, cause = dlsym(0xa5
3c8720, notAValidCommand): symbol not found`

**Comment by holzschu** [comment]
Yes. On a-Shell, you can only have 2 Python interpreters running at the same time. It's a consequence of iOS limitations (there is no fork() call, so everything is running in the same process space). I could increase the number of Python interpreters, but that would expand the size of the app. So far, this hasn't been a huge limitation, but that could change in the future.
